swarm.go

```diff
@@ -19,9 +19,8 @@ type SwarmConfig struct {
 	// `Transports.Relay` if specified.
 	DisableRelay bool `json:",omitempty"`
 
-	// EnableRelayHop makes this node act as a public relay, relaying
-	// traffic between other nodes.
-	EnableRelayHop bool
+	// DisableLimitedRelay disables the limited relay (circuit v2 relay).
+	DisableLimitedRelay bool
```

Comment:
let's call this `DisableLimitedRelayService` to disambiguate with `DisableRelay`? As it is it sounds that it disables limited relay altogether.

Reply:
or just `DisableRelayService`, shorter and probably better.

Reply:
Done.